### ISSUE TITLE
build_obb gets "stuck" on previously working facet file

### ISSUE THREAD

**Issue by bohmt** [svalinn/DAGMC#526]
build_obb  built with MOAB master now gets "stuck" trying to build the OBB tree for a facet file that previously worked with when using build_obb built with MOAB v5.0.  The facet file was originally faceted with cubit-12.2, and then converted to the new material names format using Patrick's update_dagmc_mat_names.py tool.  The facet file is a JET model with 11e6 facets and takes about 5 minutes to build the OBB tree on my computer.  The file is available to UW people at: 
https://uwmadison.box.com/s/g0ovfw8pf9u5chq0wsnqwi4pfzto32r8

**Comment by makeclean** [comment]
Don't we need an associated input deck too?

**Comment by bohmt** [comment]
I was using build_obb so I don't think I need an MCNP input file...I don't think it's worth trying to run MCNP until the tree builds.  
build_obb: A tool to prebuild your DAGMC OBB Tree
Usage: build_obb --help | [options] <dag_file>

**Comment by pshriwise** [comment]
@bohmt I pulled down this file and built OBBs successfully. Took ~4.5 min. Not sure what's going on.

Were you trying to build these OBBs on ACI?

**Comment by bohmt** [comment]
No, I did this on my computer...MOAB-master, DAGMC-develop, gcc (Ubuntu 4.8.4-2ubuntu1~14.04.3) 4.8.4

**Comment by pshriwise** [comment]
Would you mind providing the commit hash for MOAB-master?

**Comment by bohmt** [comment]
commit e30c146178faa7fa25e6bd8101ff62754848208c
Merge: e1ca947 20283a6
Author: Patrick Shriwise <shriwise@wisc.edu>
Date:   Tue Aug 22 04:22:48 2017 +0000

    Merged in pshriwise/moab/GTT_contiguous (pull request #370)
    
    GTT contiguous
    
    Approved-by: Paul Wilson <paul.wilson@wisc.edu>
    Approved-by: Vijay Mahadevan <vijay.m@gmail.com>

**Comment by gonuke** [comment]
Was someone else able to replicate this?  Is it related to the OBB tags?

**Comment by pshriwise** [comment]
I remember replicating this bug, but I cannot remember under what conditions exactly.

**Comment by gonuke** [comment]
Is this still a bug? (@pshriwise, @bohmt)  Is this workflow still relevant?
